api/attestationconfigapi/fetcher.go

```diff
@@ -71,31 +85,39 @@ func (f *fetcher) FetchLatestVersion(ctx context.Context, variant variant.Varian
 }
 
 // fetchVersionList fetches the version list information from the config API.
-func (f *fetcher) fetchVersionList(ctx context.Context, variant variant.Variant) (List, error) {
-	fetchedList, err := apifetcher.FetchAndVerify(ctx, f.HTTPClient, f.cdnURL, List{Variant: variant}, f.verifier)
+func (f *fetcher) fetchVersionList(ctx context.Context, attestationVariant Variant) (List, error) {
+	parsedVariant, err := variant.FromString(attestationVariant.String())
+	if err != nil {
+		return List{}, fmt.Errorf("parsing variant: %w", err)
+	}
+	fetchedList, err := apifetcher.FetchAndVerify(ctx, f.HTTPClient, f.cdnURL, List{Variant: parsedVariant}, f.verifier)
```

Comment:
```suggestion
func (f *fetcher) fetchVersionList(ctx context.Context, variant variant.Variant) (List, error) {
	fetchedList, err := apifetcher.FetchAndVerify(ctx, f.HTTPClient, f.cdnURL, List{Variant: variant}, f.verifier)
```
nit: functions that aren't exported can use our internal type as input. This should also save on some code duplication.

Also applies to `fetchVersion`